Makefile

```diff
@@ -99,6 +105,10 @@ ifeq ("$(BUILD_JNI)","1")
 PREQ += $(JARS)
 endif
 
+ifeq ("$(BUILD_REMOTE)", "1")
+PREQ += $(OUT)/remote/remote-worker
```

Comment:
It is not necessary to add another directory `remote`. The prefix `remote-` is self-explainable.